database/database_test.go

```diff
@@ -182,11 +181,11 @@ func TestStatesForDomains(t *testing.T) {
 	resetDB()
 
 	if err := testDB.PutState(DomainState{Name: "a.test", Status: StatusPreloaded}); err != nil {
-		t.Errorf("cannot put state %s for test StatatesForDomains", err)
+		t.Fatalf("Cannot put state a.test for test StatatesForDomains")
```

Comment:
Here and in the next one: Printint the error is fine in the Fatalf too (maybe at the end "cannot put state a.test for test StatesForDomains: %s", err)